cli/flox/doc/flox-init.md

```diff
@@ -7,39 +7,48 @@ header: "flox User Manuals"
 
 # NAME
 
-flox-init - initialize flox expressions for current project.
+flox-init - initialize a Flox environment
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] init [ `<options>` ]
+```
+flox [ <general-options> ] init
+     [ -n <name> ]
+     [ -d <path> ]
+```
 
 # DESCRIPTION
 
 Create a new empty environment in the current directory.
 
 The name of the environment will be the basename of the current directory
-or "default" if the current directory is `$HOME`.
-The `--name` flag can be used to give the environment a custom name.
+or `default` if the current directory is `$HOME`.
+The `--name` flag can be used to give the environment a specific name.
 
 By default the environment will be created in the current directory.
-flox will add a directory `$PWD/.flox`,
-within which all relevant metadata of the environment will be tracked.
+Flox will add a directory `$PWD/.flox` containing all relevant environment 
+metadata.
 The `--dir` flag can be used to create an environment in another location.
 
 If an environment already exists in the current directory,
-or the path specified using `--dir` exists, an error is returned.
+or the path specified using `--dir` exists,
+an error is returned.
 
 # OPTIONS
 
+## Init Options
+
+`-n <name>`, `--name <name>`
+:   What to name the new environment (default: current directory).
```

Comment:
Some of these you sentence case, but the search ones have lowercase first letter and no period. Any preference for which we do?